# No enterprise names

This rule forces you to properly name your classes, functions and variable.

Avoid naming things like `CatalogRepository`, `PersonFactory`, `ProductViewModel`, `ModuleFacade`.

## Rule Details

Examples of **incorrect** code when `no-enterprise` is enabled:

```javascript
class PersonsRepository {

}
```
